test/fuzzing/run-subset-fuzzer-tests.py

```diff
@@ -20,13 +39,32 @@
 print ('hb_subset_fuzzer:', hb_subset_fuzzer)
 fails = 0
 
+valgrind = None
+libtool = None
+if os.environ.get('RUN_VALGRIND', ''):
+	valgrind = which ('valgrind')
+	if valgrind is None:
+		print ("""Valgrind requested but not found.""")
+		sys.exit (1)
+	LIBTOOL = os.environ.get('LIBTOOL')
+	if LIBTOOL:
+		libtool = LIBTOOL.split(' ')[-1]
+	else:
+		libtool = which ('libtool')
+
+# Disable it for now
+valgrind = None
```

Comment:
Disabled as it fails with this
![image](https://user-images.githubusercontent.com/833473/56231369-863b4100-6093-11e9-9837-16feb639a983.png)